### PR TITLE
Feature/gradient ft

### PR DESCRIPTION
**Pull request by OzanCatalVerses**

### CODE REVIEW

File: vbgs/finetune.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+import optax
+import jax.random as jr
+import jaxsplat
+
+from vbgs.render.volume import rot_mat_to_quat, opengl_to_colmap_frame
+from vbgs.metrics import calc_psnr
+
+
+@partial(jax.jit, static_argnames=["h", "w", "c", "f"])
+def train_render(mu, scale, mat_lower, alpha, camera, h, w, c, f):
```
Comment (toonvdm) [inline vbgs/finetune.py:18]: Would be good to specify in the name that `mat_lower` is related to the rotation matrix? `lower_rot_mat` or something? 

Maybe change the order of the signature such that the splat params are grouped together?

**Review by toonvdm — APPROVED** [review]
LGTM
